LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Out of interest, why is this now required?

Reply:
No idea but required by sbt-auto-build.